helmfile.d/0100.castor.yaml

```diff
@@ -24,7 +24,14 @@ releases:
   - castor:
       noSslValidation: {{ env "NO_SSL_VALIDATION" | default false }}
       image:
-        tag: 0.1-SNAPSHOT-1576642733-7-72de814
+        registry: {{ env "CASTOR_IMAGE_REGISTRY" | default "ghcr.io" }}
+        repository: {{ env "CASTOR_IMAGE_REPOSITORY" | default "carbynestack/castor-service" }}
+        tag: {{ env "CASTOR_IMAGE_TAG" | default "0.1-SNAPSHOT-1576642733-7-72de814" }}
+        {{ if env "IMAGE_PULL_SECRET" }}
+        pullSecrets:
+          - {{ env "IMAGE_PULL_SECRET" }}
+        {{ end }}
+
```

Comment:
In order to follow a consistent formatting strategy, please remove all intermittent new lines (this one, same file below and in other files).